opi/__init__.py

```diff
@@ -310,13 +314,35 @@ def ask_keep_repo(repo):
 def print_package_names(package_names, reverse=False):
 	package_list = []
 	i = 1
+
 	for package_name in package_names:
 		package_list.append("%2d. %s" % (i, package_name))
 		i += 1
+
 	if reverse:
 		package_list.reverse()
-	for e in package_list:
-		print(e)
+
+	output_text = '\n'.join(package_list)
+
+	tmp_file = None
+
+	if which('less'):
+		try:
+			tmp_file = NamedTemporaryFile(mode='w+t', delete=False)
+			tmp_file.write(output_text)
+
+			tmp_file.close()
+
+			exit_scroller_text = ' - type q to exit the scroller'
+
+			status_text = f'(END){exit_scroller_text}:You\'re at %pB\%{exit_scroller_text}'
+
+			subprocess.call(['less', '-X', f'-P?e{status_text}', '-F', tmp_file.name])
```

Comment:
I think the empty lines here can be removed
```suggestion
			tmp_file = NamedTemporaryFile(mode='w+t', delete=False)
			tmp_file.write(output_text)
			tmp_file.close()
			exit_scroller_text = ' - type q to exit the scroller'
			status_text = f'(END){exit_scroller_text}:You\'re at %pB\%{exit_scroller_text}'
			subprocess.call(['less', '-X', f'-P?e{status_text}', '-F', tmp_file.name])
```